go.mod

```diff
@@ -8,10 +8,10 @@ require (
 	github.com/dchest/uniuri v1.2.0
 	github.com/docopt/docopt-go v0.0.0-20180111231733-ee0de3bc6815
 	github.com/dustin/go-humanize v1.0.1
-	github.com/elastic/crd-ref-docs v0.0.10
+	github.com/elastic/crd-ref-docs v0.0.12
```

Comment:
Update here also https://github.com/fission/fission/blob/241983d2f7873eca18e91fa80218eb0025f45a31/Makefile#L103

Reply:
Please ensure you run `make all-generators` command and push all changed files along with these modules update.

Reply:
Done